src/rainmaker/view/item.ts

```diff
@@ -48,8 +48,9 @@ export class RMakerItem extends TreeItem {
   public set commandId(v: string) {
     this.command = {
       command: v,
-      title: this.label,
-      tooltip: this.tooltip,
+      title: typeof this.label === "string" ? this.label : this.label.label,
+      tooltip:
+        typeof this.tooltip === "string" ? this.tooltip : this.tooltip.value,
```

Comment:
This change is causing crash if user is not logged in already

Reply:
I'm not sure why would this crash the rainmaker tree view. Any additional information or related log ?

Reply:
if `this.tooltip == undefined` then it will crash because `.value` on `undefined`